report/db.py

```diff
@@ -0,0 +1,9 @@
+import sqlalchemy
+from sqlalchemy.orm import sessionmaker
+
+SESSION = sessionmaker()
+
+
+def make_engine(settings):
+    """Construct a sqlalchemy engine from the passed ``settings``."""
+    return sqlalchemy.create_engine(settings["sqlalchemy.url"])
```

Comment:
Can we have this be `database_url`? Instead of `sqlalchemy.url`. I think I prefer the environment variables and these names being the same like they are in LMS. I think this is an `h` quirk?

In LMS (for the most part) you can get the internal name by lower casing the environment variable name.